fuzz/fuzz_targets/packet.rs

```diff
@@ -0,0 +1,29 @@
+#![cfg_attr(not(windows), no_main)]
+
+#[cfg(not(windows))]
+use libfuzzer_sys::fuzz_target;
+
+#[cfg(not(windows))]
+fuzz_target!(|data: &[u8]| {
+    use std::sync::Once;
+
+    use neqo_transport::{packet::PublicPacket, RandomConnectionIdGenerator};
+
+    static INIT: Once = Once::new();
+    static mut DECODER: Option<neqo_transport::RandomConnectionIdGenerator> = None;
+
+    // Initialize things
+    unsafe {
+        INIT.call_once(|| {
+            neqo_crypto::init();
+            DECODER = Some(RandomConnectionIdGenerator::new(20));
```

Comment:
Instead of the use of `unsafe`, how about using `lazy_static!`. (Note that we already have the dependency in our tree, thus no additional compile time or attack vector.)

``` diff
modified   fuzz/Cargo.toml
@@ -16,6 +16,7 @@ libfuzzer-sys = { version = "0.4" }
 neqo-common = { path = "../neqo-common" }
 neqo-crypto = { path = "../neqo-crypto" }
 neqo-transport = { path = "../neqo-transport", features = ["fuzzing"] }
+lazy_static = "1.4.0"
 
 [lints]
 workspace = true
modified   fuzz/fuzz_targets/packet.rs
@@ -1,22 +1,16 @@
 #![no_main]
-use std::sync::Once;
+use lazy_static::lazy_static;
 
 use libfuzzer_sys::fuzz_target;
 use neqo_transport::{packet::PublicPacket, RandomConnectionIdGenerator};
 
-static INIT: Once = Once::new();
-static mut DECODER: Option<neqo_transport::RandomConnectionIdGenerator> = None;
+lazy_static! {
+    static ref DECODER: RandomConnectionIdGenerator = RandomConnectionIdGenerator::new(20);
+}
 
 fuzz_target!(|data: &[u8]| {
-    // Initialize things
-    unsafe {
-        INIT.call_once(|| {
-            neqo_crypto::init();
-            DECODER = Some(RandomConnectionIdGenerator::new(20));
-        });
-    }
-    let decoder = unsafe { DECODER.as_ref().unwrap() };
+    neqo_crypto::init();
 
     // Run the fuzzer
-    let _ = PublicPacket::decode(data, decoder);
+    let _ = PublicPacket::decode(data, &*DECODER);
 });
```